resources/assets/less/bem/beatmap-scoreboard-table.less

```diff
@@ -20,10 +20,27 @@
   @_top: beatmap-scoreboard-table;
   @row-height: 1.8em;
 
-
   width: 100%;
   overflow-x: auto;
 
+  .row-colour(@colour) {
+    td {
+      background-color: lighten(@colour, 20%);
+    }
+
+    &:nth-child(odd) {
+      td {
+        background-color: lighten(@colour, 20%);
```

Comment:
different color in case a bunch of friends ranked consecutively? (in which case this function can be used for default color as well)

Reply:
It looks weird if there are different colours and stuff isn't consecutive - you get stuff like this:
![](https://puu.sh/zndYf/3923a12ba0.png)

which just looks wrong imo